db/migrate/20170508215743_create_shelters.rb

```diff
@@ -0,0 +1,10 @@
+class CreateShelters < ActiveRecord::Migration[5.1]
+  def change
+    create_table :shelters do |t|
+      t.string :name, null: false
+      t.string :subdomain, null: false
```

Comment:
I know there won't be much data in this table, but maybe it would be good to add index here anyway as this column will be used in filtering queries often, what do you think?